docs/site/Discovering-models-from-relational-databases.md

```diff
@@ -0,0 +1,43 @@
+---
+lang: en
+title: "Discovering models from relational databases"
+keywords: LoopBack 4.0, LoopBack-Next
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Discovering-models-from-relational-databases.html
+---
+
+## Synopsis
+
+LoopBack makes it simple to create models from an existing relational database.
+This process is called _discovery_ and is supported by the following connectors:
+
+* [Cassandra](Cassandra-connector.html)
+* [MySQL](MySQL-connector.html)
+* [Oracle](Oracle-connector.html)
+* [PostgreSQL](PostgreSQL-connector.html)
+* [SQL Server](SQL-Server-connector.html)
+* [IBM DB2](DB2-connector.html)
+* [IBM DashDB](DashDB.html)
+* [IBM DB2 for z/OS](DB2-for-z-OS.html)
+* [SAP HANA](https://www.npmjs.org/package/loopback-connector-saphana) - Not officially supported; see [Community connectors](Community-connectors.html).
+
+
+## Overview
+Models can be discovered from a supported datasource by running the `lb4 discover` command.
+
+**The loopback project must be built and contain the built datasource files in `PROJECT_DIR/dist/src/datasources.js**
+
+### Options
+
+`--dataSource`: Put a valid datasource name here to skip the datasource prompt
+
+`--views`: Choose whether to discover views, default true
+
+`--all`: Skips the model prompt and discovers all of them
+
+`--outDir`: Specify the directory into which the `model.model.ts` files will be placed
+
+`--schema`: Specify the schema which the datasource will find the models to discover
+
+`--schemaDefs`: The path to a `.json` file containing a model definition, like those used loopback 2 and 3.  Used to make a typescript model out of an old loopback model without any prompts.
```

Comment:
- `loopback` -> `LoopBack`
- For the second sentence, I'm wondering if it would be better:
   ```
   LoopBack 4 models will be created out of the LoopBack 3 models specified in `.json` file without any prompts.
   ```

Reply:
Significantly better, fixed.